component/src/main/java/me/androidbox/component/general/AgendaBottonSheetToolbar.kt

```diff
@@ -0,0 +1,74 @@
+package me.androidbox.component.general
+
+import androidx.annotation.StringRes
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import me.androidbox.component.R
+import me.androidbox.component.ui.theme.Black
+import me.androidbox.component.ui.theme.BusbyTaskyTheme
+import me.androidbox.component.ui.theme.White
+import me.androidbox.component.ui.theme.agendaBodyTextColor
+
+@Composable
+fun AgendaBottomSheetToolbar(
+    @StringRes title: Int,
```

Comment:
I'd rather pass a string here, that keeps it flexible if you pass in a string resource or a dynamic string

Reply:
@philipplackner updated